vad_transcriber/audioTranscript_gui.py

```diff
@@ -119,7 +126,11 @@ def initUI(self):
         self.transcribeWav.setToolTip('Start Wav Transcription')
         self.openMicrophone = QPushButton('Start Speaking', self)
         self.openMicrophone.setToolTip('Open Microphone')
-
+        
+        #
+        self.CheckConvertSR = QPushButton('Check_Convert_SR', self)
```

Comment:
Why adding an extra option and not just handling that in the existing transcription process?